webmessaging/message-channels/close.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+async_test(t => {
+    const c = new MessageChannel();
+    c.port1.onmessage = t.unreached_func('Should not have delivered message');
+    c.port1.close();
+    c.port2.postMessage('TEST');
+    setTimeout(t.step_func_done(), 100);
```

Comment:
It doesn't really rely on anything in the spec, no. I made it a constant now.

I don't think there really is a way to make this more robust (but at least the worst that can happen is that the test passes when it shouldn't). MessagePorts are specifically defined with each port having its own task source (https://html.spec.whatwg.org/multipage/comms.html#port-message-queue). And browser are free to use separate queues for separate task sources (https://html.spec.whatwg.org/multipage/webappapis.html#task-source) (in fact in the near future Chrome is likely going to start relying on that).